tests/analyzer/test_baseline.json

```diff
@@ -1,9 +1,33 @@
 {
```

Comment:
General question: Why do we need to change previous test? Why not add the new test for features? We always need to add feature, and it would be great if we don't change previous test logic.

Reply:
Solved